tests/common/utils_unit_test.cc

```diff
@@ -0,0 +1,29 @@
+#include "gtest/gtest.h"
+#include "src/common/utils.h"
+
+using namespace gfs::common::utils;
+
+class UtilsUnitTest : public ::testing::Test {
+};
+
+// Test different cases for checking the validity of a pathname
+TEST_F(UtilsUnitTest, ValidPathnameTest) {
+   auto emptyPathnameRes(checkPathnameValidity(""));
+   EXPECT_EQ(emptyPathnameRes.error_message(), 
```

Comment:
We should **never** test error messages themselves. Instead, as described in above, we should set the error code correctly (`NOT_FOUND`, `ALREADY_EXISTS`, `FAILED_PRECONDITION`, etc.) and test the error codes only.

Reply:
This will get addressed in a followup.